.github/workflows/ci.yml

```
@@ -0,0 +1,55 @@
+name: CI
+on:
+  push:
+    branches: ["master", "src-layout"]
+  pull_request:
+    branches: ["master", "src-layout"]
+  schedule:
+    - cron: "0 12 1 * *"
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  generate-compatibility-matrix:
+    name: >-
+      Check for any uncommitted
+      table alteration
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Determine Python versions
+      run: |
+        {
+          echo 'PYTHON_VERSIONS<<EOF'
```

Shouldn't this be
```suggestion
          echo 'PYTHON_VERSIONS=<<EOF'
```
?

---

Apparently, the _cmd_ for a Bash heredoc expression can also be a variable name.

---

What do you mean? You can use `shell: bash` and that should get rid of the syntactic differences between the runtimes...

---

I was mistaken, it's actually [GH-specific syntax](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#example-of-a-multiline-string).

---

Oh, I didn't realize. I thought this was a normal bash here-string..

---

Marking as resolved.